pkg/config/config_suite_test.go

```diff
@@ -27,3 +32,55 @@ func TestConfig(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecs(t, "os2 Config Suite")
 }
+
+var _ = Describe("os2 config unit tests", func() {
```

Comment:
os2 ?!

Reply:
woops, copied from the test_env which also has the os2 stuff , will change it in both.